api/api/specs/routers.yaml

```diff
@@ -793,14 +793,7 @@ components:
           type: "string"
 
     EnvVar:
```

Comment:
Same, don't need to create a local copy of `EnvVar` schema

Reply:
done